packages/paste-core/utilities/text/src/index.tsx

```diff
@@ -1,9 +1,9 @@
 import styled from '@emotion/styled';
 import {compose, space, display, verticalAlign, overflow, typography, system} from 'styled-system';
-import {SpaceProps, Display, VerticalAlign, OverflowProps, TypographyProps} from '@twilio-paste/types';
+import {SpaceProps, Display, VerticalAlign, OverflowProps, TypographyProps} from '@twilio-paste/style-props';
 
-interface Text extends SpaceProps, OverflowProps, TypographyProps {
-  as?: keyof JSX.IntrinsicElements;
+interface Text extends React.AllHTMLAttributes<''>, SpaceProps, OverflowProps, TypographyProps {
```

Comment:
Need to allow basic HTML attributes on `Text`, stuff like id doesn't come for free.

Reply:
This may or may not need to be `React.AllHTMLAttributes<any>` as it is literally _any element_ that text can be. Just not sure if `''` is _proper_ typescript. It works 🤷‍♂